zebra/zapi_msg.c

```diff
@@ -1084,6 +1085,29 @@ static int zsend_table_manager_connect_response(struct zserv *client,
 	return zserv_send_message(client, s);
 }
 
+static int zsend_srv6_manager_connect_response(struct zserv *client,
+						vrf_id_t vrf_id,
+						uint16_t result)
+{
+	struct stream *s = stream_new(ZEBRA_MAX_PACKET_SIZ);
+
+	zclient_create_header(s, ZEBRA_SRV6_MANAGER_CONNECT, vrf_id);
+
+	/* proto */
+	stream_putc(s, client->proto);
+
+	/* instance */
+	stream_putw(s, client->instance);
```

Comment:
why send proto and instance back to the client? the client set these when it brought the zapi session up.

Reply:
This is the same topic as the previous comment.

ref: zsend_label_manager_connect_response
https://github.com/FRRouting/frr/blob/f79d311b3e0ea744cad115978b10bad2661afea9/zebra/zapi_msg.c#L1035-L1039

Reply:
Already fixed.